docs/content/en/docs/Reference/reset.md

```diff
@@ -20,7 +111,7 @@ It is possible to reset the state of a node by either booting into the "Reset" m
 
 It is possible to trigger the reset manually by logging into the recovery from the boot menu and running `kairos reset` from the console.
```

Comment:
The sections in the description at the top don't match the sections below:

```
via the Boot menu, executed manually, or triggered via Kubernetes
```

and the sections:

- Remotely, via command line
- From Kubernetes
- From the boot menu
- Manual reset

I would change the order to match the description and add the "manual reset" in the description too.

Reply:
good point, fixed!